IBPSA/Fluid/FixedResistances/PlugFlowPipe.mo

```diff
@@ -151,25 +128,132 @@ protected
     "Default density (e.g., rho_liquidWater = 995, rho_air = 1.2)"
     annotation (Dialog(group="Advanced"));
 
+  // In the volume, below, we scale down V and use
+  // mSenFac. Otherwise, for air, we would get very large volumes
+  // which affect the delay of water vapor and contaminants.
+  // See also IBPSA.Fluid.FixedResistances.Validation.PlugFlowPipes.TransportWaterAir
+  // for why mSenFac is 10 and not 1000, as this gives more reasonable
+  // temperature step response
+  Fluid.MixingVolumes.MixingVolume vol(
+    redeclare final package Medium = Medium,
+    final m_flow_nominal=m_flow_nominal,
+    final V=if rho_default > 500 then VEqu else VEqu/1000,
+    final nPorts=2,
+    final T_start=T_start_out,
+    final energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    final mSenFac=if rho_default > 500 then 1 else 10) if have_pipCap
+    "Control volume connected to port_b. Represents equivalent pipe wall thermal capacity."
+    annotation (Placement(transformation(extent={{70,20},{90,40}})));
+
+  LosslessPipe noMixPip(
```

Comment:
why not use a simple connection instead?

Reply:
I believe I've written it like that to conserve a symmetrical syntax between the case with the mixing volume (`have_pipCap`) and the case without a mixing volume (`not have_pipCap`).
Also, in the graphical editor of Dymola, you get an annotation in the hover tooltip when a component is optional (screenshot below), but not when a connect statement is optional, which makes that structure a bit more explicit.
![conditionalPip](https://user-images.githubusercontent.com/48033702/134583201-04ac4d6e-e65d-4c95-b0d5-6e270aba271e.jpg)

That being said, if there is a computational or syntax issue with that format, I can switch back to an optional connect.